README.md

```diff
@@ -113,6 +113,92 @@ In case of using buttons that have icons in them, make sure to include font awes
 ```
 <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css" integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc" crossorigin="anonymous">
 ```
+# Base Icon Buttons
+
+The `base-icon-btn` class enables you to have a normal, customizable button. Using it, you decide to add any fontawesome icon in it. You can choose to either place it at the left or right side of the button with the available classes which you will see below. For the icon to be positioned well in the button, you should place the button text after the fontawesome icon if you want the icon to be on the left side of the button and vice versa for the right icon too.
+
+- ## Usage
+
+Examples
+- [Base buttons](#base-icon-button)
+- [Rounded buttons](#rounded-buttons)
+- [Block buttons](#block-icon-buttons)
+
+* ### base icon button 
+ The `base-icon-btn class`, gives you a default button with no background color and a border. You can choose to style it by adding, some of the other available classes, like `btn-orange` to change the background color, and `icon-right` to position the icon in the button. You can make use of it, by copying the sinppet below.
```

Comment:
class should be outside the `

Reply:
"You can choose to style it by adding some ..." remove the comma after adding

Reply:
Ok, thank you for noting this.

Reply:
I've  resolved that